hackathon_site/hardware/test_api.py

```diff
@@ -598,7 +627,34 @@ def test_search_filter(self):
         results = data["results"]
 
         returned_ids = [res["id"] for res in results]
-        self.assertCountEqual(returned_ids, [self.order_3.id])
+        self.assertCountEqual(returned_ids, [self.order_3.id, self.order_4.id])
+
+    def test_status_filter_with_only_view_permissions(self):
+        Profile.objects.create(user=self.user, team=self.team)
+        self._login(self.view_permissions)
+
+        url = self._build_filter_url(status="Submitted")
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        results = data["results"]
+
+        returned_ids = [res["id"] for res in results]
+        self.assertCountEqual(returned_ids, [self.order_2.id])
+
+    def test_filter_from_another_team_with_only_view_permissions(self):
```

Comment:
Again, if a user has view permissions, they should get access to everything - that's what tech team will have.

And as above, we know how the team code filter works, so this shouldn't actually require a test